ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -10228,6 +10228,14 @@ ip as-path access-list mylist2 deny _64517$ igp
 | IPv4 Access-list | ACL |
 | Start limit | Infinite |
 
+#### 802.1X VLAN Assignment Groups
```

Comment:
feels like we are missig doc for the dropped statistics and the radius av-pairs

Reply:
given that we don't have a table for all the radius av-pair I suggest we open a new issue for just this table to track it.

for the dropped statistic lets add a column in the global table

Reply:
Fixed!
And raised issue.